packit/api.py

```diff
@@ -2343,11 +2347,37 @@ def run_osh_build(
         if csmock_args:
             cmd.append("--csmock-args=" + shlex.quote(csmock_args))
 
-        cmd.append("--config=" + str(chroot))
+        # osh_options = self.package_config.osh_options.__dict__
+
+        # if osh_options is not None:
+        #     analyzer = osh_options.get("analyzer")
+        #     config = osh_options.get("config")
+        #     profile = osh_options.get("profile")
+
+        osh_config = str(chroot)
+
+        if osh_options := self.package_config.osh_options.__dict__:
+            if analyzer := osh_options.get("analyzer"):
+                cmd.append("--analyzer=" + shlex.quote(analyzer))
+            if profile := osh_options.get("profile"):
+                cmd.append("--profile=" + shlex.quote(profile))
+            if config := osh_options.get("config"):
+                osh_config = shlex.quote(config)
```

Comment:
As for these, you have created a wrapper class for those, so there's no need to access the dictionaries directly:

```suggestion
            if analyzer := osh_options.analyzer:
                cmd.append("--analyzer=" + shlex.quote(analyzer))
            if profile := osh_options.profile:
                cmd.append("--profile=" + shlex.quote(profile))
            if config := osh_options.config:
                osh_config = shlex.quote(config)
```